test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1

```diff
@@ -0,0 +1,890 @@
+﻿##
+## Tests for PowerShell system lock down and constrained language mode restrictions
+## These are Windows platform only tests
+##
+
+function Get-RandomFileName

```

Comment:
this seems like a mighty useful function, perhaps it should be in the test/tools/Modules/HelpersCommon modules

Reply:
Will do.